File: src-tauri/src/cmd/menu.rs
use tauri::command;

use crate::types::menu::Menu; 

#[command]
pub fn get_menu_list(page_id: String) -> Result<(), tauri::Error> {
    Ok(())
}
// menu
#[command]
pub fn add_menu(item: Menu ) ->Result<(), tauri::Error> {
    Ok(())
}

#[command]
pub fn update_menu(page_id: String) -> Result<(), tauri::Error> {
    Ok(())
}

#[command]
pub fn delete_menu(page_id: String) -> Result<(), tauri::Error> {
    Ok(())
}

#[command]
pub fn copy_menu(page_id: String) -> Result<(), tauri::Error> {
    Ok(())
}

File: packages/editor/src/invokeApi/menu.ts
import { invoke } from "@tauri-apps/api/core";

// 获取菜单列表
export const getMenuList = (params: any) => {
  return invoke("get_menu_list", params)
};

// 新增菜单
export const addMenu = (params: any) => {
  return invoke("add_menu", params)
};

// 删除菜单
export const delMenu = (params: any) => {
  return invoke("delete_menu", params)
};

// 更新菜单
export const updateMenu = (params: any) => {
  return invoke("update_menu", params)
};

// 复制菜单
export const copyMenu = (params: { projectId: string, id : string }) => {
  return invoke("copy_menu", params)
};


